README.md

```diff
@@ -32,19 +32,31 @@ This guide executes a Sonobuoy run on your cluster, and records the following re
 
 * *You should have `kubectl` installed.* If not, follow the instructions for [installing via Homebrew (MacOS)][6] or [building the binary (Linux)][7].
 
+* *You should have `docker` installed (minimum 17.05).* Docker images are used to generate the YAML examples, based on your cluster's RBAC settings.
```

Comment:
ah ok, thanks for the heads up---the ksonnet PR is on hold til Alex gets back tomorrow, but if I finish the Ark PR, I'll come back and try to take out that dependency